### ISSUE TITLE
Positive Lookbehind leads to bad state: no element

### ISSUE THREAD

**Issue by Decron** [micazi/rich_text_controller#39]
This error is thrown as soon as a positive lookahead / lookbehind completes.

My setup is as follows:
```
static final LinkedHashMap<RegExp, TextStyle> patternUser = LinkedHashMap.of({
    RegExp(r"(?<=@italic )hello", multiLine: true): TextStyle(color: Colors.grey, fontStyle: FontStyle.italic),
});

   RichTextController _controller = RichTextController(
     patternMatchMap: patternUser,
     onMatch: (List<String> match) {

        if (match[0] != null) {
            print("Match 0 - ${match[0]}");
        } else if (match[1] != null) {
            print("Match 1 - ${match[1]}");
        }

     },
    regExpMultiLine: true,
    regExpUnicode: true,
    regExpDotAll: true,
    );

TextFormField(
              textInputAction: TextInputAction.newline,
              keyboardType: TextInputType.multiline,
              maxLines: null,
              focusNode: focusNode,
              controller: controller,
              decoration: InputDecoration(
                hintText: 'Type your message...',
                contentPadding: const EdgeInsets.symmetric(horizontal: 10, vertical: 5),
                border: OutlineInputBorder(
                  borderRadius: BorderRadius.circular(20),
                  borderSide: BorderSide.none,
                ),
                filled: true,
              ),
              style: TextStyle(
                fontSize: 14,
                height: 1.5,
                color: Colors.black.withOpacity(0.9),
              ),
            )

```
For instance a test string will fail once the user enter `"test @italics hello"`.

Here is the full error message:

The following StateError was thrown building Scrollable-[GlobalKey#40827](axisDirection: down, physics: null, restorationId: "editable", dependencies: [MediaQuery, UnmanagedRestorationScope, _InheritedTheme, _LocalizationsScope-[GlobalKey#2aa32]], state: ScrollableState#7b964(position: ScrollPositionWithSingleContext#f8918(offset: 0.0, range: 0.0..0.0, viewport: 21.0, ScrollableState, BouncingScrollPhysics -> RangeMaintainingScrollPhysics -> BouncingScrollPhysics -> RangeMaintainingScrollPhysics, IdleScrollActivity#906d6, ScrollDirection.idle), effective physics: BouncingScrollPhysics -> RangeMaintainingScrollPhysics -> BouncingScrollPhysics -> RangeMaintainingScrollPhysics)):
Bad state: No element

The relevant error-causing widget was: 
  TextFormField TextFormField:<Decron'sFilePath>:44:18
When the exception was thrown, this was the stack: 
#0      Iterable.firstWhere (dart:core/iterable.dart:700:5)
#1      RichTextController.buildTextSpan.<anonymous closure> (package:rich_text_controller/rich_text_controller.dart:139:52)
#2      _StringBase.splitMapJoin (dart:core-patch/string_patch.dart:818:27)
#3      RichTextController.buildTextSpan (package:rich_text_controller/rich_text_controller.dart:115:10)
#4      EditableTextState.buildTextSpan (package:flutter/src/widgets/editable_text.dart:4789:30)
#5      EditableTextState.build.<anonymous closure> (package:flutter/src/widgets/editable_text.dart:4674:41)
#6      ScrollableState.build (package:flutter/src/widgets/scrollable.dart:966:44)
#7      StatefulElement.build (package:flutter/src/widgets/framework.dart:5409:27)
#8      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5297:15)
#9      StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#10     Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#11     StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#12     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#13     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#14     Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#15     ProxyElement.update (package:flutter/src/widgets/framework.dart:5628:5)
#16     _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:107:11)
#17     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#18     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#19     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#20     Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#21     StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#22     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#23     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#24     Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#25     ProxyElement.update (package:flutter/src/widgets/framework.dart:5628:5)
#26     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#27     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#28     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#29     Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#30     StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#31     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#32     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#33     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#34     Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#35     StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#36     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#37     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#38     Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#39     ProxyElement.update (package:flutter/src/widgets/framework.dart:5628:5)
#40     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#41     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#42     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#43     Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#44     StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#45     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#46     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#47     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#48     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#49     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#50     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#51     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#52     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#53     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#54     Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#55     StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#56     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#57     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#58     Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#59     ProxyElement.update (package:flutter/src/widgets/framework.dart:5628:5)
#60     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#61     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#62     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#63     SlottedRenderObjectElement._updateChildren (package:flutter/src/widgets/slotted_render_object_widget.dart:295:33)
#64     SlottedRenderObjectElement.update (package:flutter/src/widgets/slotted_render_object_widget.dart:256:5)
#65     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#66     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#67     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#68     Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#69     StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#70     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#71     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#72     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#73     Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#74     StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#75     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#76     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#77     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#78     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#79     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#80     Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#81     StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#82     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#83     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#84     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#85     Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#86     StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#87     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#88     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#89     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#90     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#91     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#92     Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#93     StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#94     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#95     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#96     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#97     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#98     Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#99     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#100    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#101    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#102    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#103    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#104    StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#105    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#106    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#107    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#108    ProxyElement.update (package:flutter/src/widgets/framework.dart:5628:5)
#109    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#110    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#111    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#112    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#113    StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#114    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#115    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#116    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#117    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#118    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#119    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#120    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#121    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#122    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#123    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#124    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#125    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#126    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#127    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#128    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#129    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#130    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#131    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#132    StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#133    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#134    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#135    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#136    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#137    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#138    ProxyElement.update (package:flutter/src/widgets/framework.dart:5628:5)
#139    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#140    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#141    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#142    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#143    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#144    ProxyElement.update (package:flutter/src/widgets/framework.dart:5628:5)
#145    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#146    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#147    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#148    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#149    StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#150    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#151    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#152    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#153    ProxyElement.update (package:flutter/src/widgets/framework.dart:5628:5)
#154    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#155    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#156    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#157    StatelessElement.update (package:flutter/src/widgets/framework.dart:5373:5)
#158    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#159    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#160    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#161    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#162    StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#163    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#164    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#165    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#166    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#167    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#168    StatelessElement.update (package:flutter/src/widgets/framework.dart:5373:5)
#169    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#170    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#171    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#172    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#173    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#174    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#175    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#176    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#177    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#178    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#179    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#180    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#181    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#182    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#183    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#184    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#185    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#186    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#187    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#188    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#189    StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#190    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#191    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#192    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#193    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#194    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#195    ProxyElement.update (package:flutter/src/widgets/framework.dart:5628:5)
#196    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#197    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#198    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#199    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#200    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#201    ProxyElement.update (package:flutter/src/widgets/framework.dart:5628:5)
#202    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#203    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#204    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#205    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#206    StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#207    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#208    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#209    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#210    StatelessElement.update (package:flutter/src/widgets/framework.dart:5373:5)
#211    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#212    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#213    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#214    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#215    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#216    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#217    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#218    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#219    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#220    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#221    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#222    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#223    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#224    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#225    StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#226    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#227    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#228    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#229    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#230    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#231    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#232    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#233    ProxyElement.update (package:flutter/src/widgets/framework.dart:5628:5)
#234    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#235    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#236    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#237    ProxyElement.update (package:flutter/src/widgets/framework.dart:5628:5)
#238    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#239    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#240    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5462:11)
#241    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#242    StatefulElement.update (package:flutter/src/widgets/framework.dart:5485:5)
#243    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#244    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#245    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#246    StatelessElement.update (package:flutter/src/widgets/framework.dart:5373:5)
#247    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#248    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#249    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#250    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#251    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#252    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#253    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#254    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6441:14)
#255    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#256    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5322:16)
#257    Element.rebuild (package:flutter/src/widgets/framework.dart:5016:7)
#258    StatelessElement.update (package:flutter/src/widgets/framework.dart:5373:5)
#259    Element.updateChild (package:flutter/src/widgets/framework.dart:3685:15)
#260    _LayoutBuilderElement._layout.layoutCallback (package:flutter/src/widgets/layout_builder.dart:135:18)
#261    BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2719:19)
#262    _LayoutBuilderElement._layout (package:flutter/src/widgets/layout_builder.dart:153:12)
#263    RenderObject.invokeLayoutCallback.<anonymous closure> (package:flutter/src/rendering/object.dart:2604:59)
#264    PipelineOwner._enableMutationsToDirtySubtrees (package:flutter/src/rendering/object.dart:1059:15)
#265    RenderObject.invokeLayoutCallback (package:flutter/src/rendering/object.dart:2604:14)
#266    RenderConstrainedLayoutBuilder.rebuildIfNecessary (package:flutter/src/widgets/layout_builder.dart:228:7)
#267    _RenderLayoutBuilder.performLayout (package:flutter/src/widgets/layout_builder.dart:313:5)
#268    RenderObject.layout (package:flutter/src/rendering/object.dart:2493:7)
#269    RenderBox.layout (package:flutter/src/rendering/box.dart:2382:11)
#270    ChildLayoutHelper.layoutChild (package:flutter/src/rendering/layout_helper.dart:52:11)
#271    RenderFlex._computeSizes (package:flutter/src/rendering/flex.dart:868:45)
#272    RenderFlex.performLayout (package:flutter/src/rendering/flex.dart:903:32)
#273    RenderObject.layout (package:flutter/src/rendering/object.dart:2493:7)
#274    RenderBox.layout (package:flutter/src/rendering/box.dart:2382:11)
#275    RenderStack.layoutPositionedChild (package:flutter/src/rendering/stack.dart:511:11)
#276    RenderStack.performLayout (package:flutter/src/rendering/stack.dart:620:30)
#277    RenderObject.layout (package:flutter/src/rendering/object.dart:2493:7)
#278    RenderBox.layout (package:flutter/src/rendering/box.dart:2382:11)
#279    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:104:21)
#280    RenderObject.layout (package:flutter/src/rendering/object.dart:2493:7)
#281    RenderBox.layout (package:flutter/src/rendering/box.dart:2382:11)
#282    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:104:21)
#283    RenderObject.layout (package:flutter/src/rendering/object.dart:2493:7)
#284    RenderBox.layout (package:flutter/src/rendering/box.dart:2382:11)
#285    MultiChildLayoutDelegate.layoutChild (package:flutter/src/rendering/custom_layout.dart:173:12)
#286    _ScaffoldLayout.performLayout (package:flutter/src/material/scaffold.dart:1056:7)
#287    MultiChildLayoutDelegate._callPerformLayout (package:flutter/src/rendering/custom_layout.dart:237:7)
#288    RenderCustomMultiChildLayoutBox.performLayout (package:flutter/src/rendering/custom_layout.dart:405:14)
#289    RenderObject._layoutWithoutResize (package:flutter/src/rendering/object.dart:2332:7)
#290    PipelineOwner.flushLayout (package:flutter/src/rendering/object.dart:1013:18)
#291    RendererBinding.drawFrame (package:flutter/src/rendering/binding.dart:494:19)
#292    WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:918:13)
#293    RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:360:5)
#294    SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1297:15)
#295    SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1227:9)
#296    SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:1085:5)
#297    _invoke (dart:ui/hooks.dart:170:13)
#298    PlatformDispatcher._drawFrame (dart:ui/platform_dispatcher.dart:401:5)
#299    _drawFrame (dart:ui/hooks.dart:140:31)

**Comment by Decron** [comment]
My short-term hack is to leverage negative lookbehind and lookaheads, which still work.

1. change the default text style to what you would like the the "positive lookbehind" pattern to catch.
2. Make a negative lookbehind which catches normal text and styles it to what you actually want your default to be.
For instance, the corresponding negative lookbehind for my original pattern above would be `RegExp(r'(?<!@ italic\s)\b\w+\b')`

**Comment by micazi** [comment]
@Decron Thank you for your feedback, I will look into it this week and figure out a solution.

**Comment by GameTec-live** [comment]
any ETA on when this will be fixed? makes the library unusable for me...

**Comment by micazi** [comment]
Closed and updated in version 2.0.0
Sorry, I haven't been active lately.
Thank you for your support!
